### DISCUSSION TITLE
RIP Jacob Myers aka Jyers.

### DISCUSSION THREAD

**Discussion by dinosaurusrex39** [Jyers/Marlin#1752] [General Discussion]
Thank you Jyers for giving so many of us Ender 3 v2 owners life into our printers. May you Rest In Peace. We will continue what you started.

**Comment by doubleO00** [comment]
Dude what are you saying ????

**Reply by zedn4c3r** [reply]
Could you please provide your source? The regulars in general on discord are very concerned.

**Comment by method1020** [comment]
User just signed up for Github, made a Marlin repository, and then commented this. I am fairly certain this is a troll post.

**Comment by Jyers** [comment]
To clarify I am in fact still asking the living 😁

Just perpetually busy, although I should have time soon for some updates 👍

**Comment by JotWee** [comment]
User blocked…
